docs/design/kademlia-removal.md

```diff
@@ -0,0 +1,84 @@
+# Kademlia Removal
+
+## Abstract
+
+This design document outlines the communication protocol between satellites and 
+storage nodes, network refreshes, and kademlia removal given the satellite opt-in 
+capability for storage nodes. 
+
+## Background
+User research on how Storage Node Operators would connect to satellites resulted in a 
+simple solution called Opt-in SNO select. Satellites wait for storage nodes to work 
```

Comment:
Storage Nodes select which Satellites to work with, hence the Storage Nodes do not need to discover Satellites on the network and Satellites have no benefit from finding Storage Nodes on the network.